tests/cypress/integration/issue_1785_propagation_latest_frame.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check propagation work from the latest frame', () => {
+
+    const issueId = '1785'
+    const labelName = `Issue ${issueId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const images = [`image_${issueId}_1.png`,
+                    `image_${issueId}_2.png`,
+                    `image_${issueId}_3.png`]
+    const width = 800
+    const height = 800
+    const posX = 10
+    const posY = 10
+    const color = 'gray'
+    const archiveName = `images_issue_${issueId}.zip`
+    const archivePath = `cypress/fixtures/${archiveName}`
+    const imagesFolder = `cypress/fixtures/image_issue_${issueId}`
+    const lastElement = images[images.length - 1]
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        for (let img of images) {
+            cy.imageGenerator(imagesFolder, img, width, height, color, posX, posY, labelName)
+        }
+        cy.wait(2000)
+        cy.createZipArchive(imagesFolder, archivePath)
+        cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName)
+        cy.openTaskJob(taskName)
+    })
+
+    describe(`Testing issue "${issueId}"`, () => {
+        it('Go to the last frame', () => {
+            cy.get('.cvat-annotation-header')
+            .find(':nth-child(7)')
```

Comment:
Can we escape getting a child using this selector? It isn't also really reliable to get element using a child number

Reply:
Unfortunately, there is nothing to focus on. Can just take the last element, but if you change the button, the test will stop working. It would be nice to attach different classes to these elements. As well as panel elements .cvat-canvas-controls-sidebar

Reply:
We have added selectors for you: https://github.com/opencv/cvat/pull/2038

Reply:
Thank you very much.